CHANGES.txt

```diff
@@ -1,5 +1,12 @@
 CHANGES
 
+2.1.0
+- Localmode to support reload features.
+- Refactor InMemory Metrics and Impressions to comply cache interface style.
+- Support for Redis cache and Redis client.
+- Configuration to support multiple modes and Redis parameters.
+- Support for multiple splits evaluation by including GetTreatments method.
+
```

Comment:
The only added option is "Mode", and its default is ´Standalone´. So user don't need to make it explicit. (no incompatible change)